packages/multi-column/src/test/conversions/__snapshots__/multi-column/undefined/external.html

```diff
@@ -0,0 +1 @@
+<div class="bn-block-column-list" data-node-type="columnList" data-id="1"></div><div class="bn-block-column" data-node-type="column" data-id="2" data-width="1" style="flex-grow: 1;"></div><div class="bn-block-outer" data-node-type="blockOuter" data-id="3"><div class="bn-block" data-node-type="blockContainer" data-id="3">Column Paragraph 0</div></div><div class="bn-block-outer" data-node-type="blockOuter" data-id="4"><div class="bn-block" data-node-type="blockContainer" data-id="4">Column Paragraph 1</div></div><div class="bn-block-column" data-node-type="column" data-id="5" data-width="1" style="flex-grow: 1;"></div><div class="bn-block-outer" data-node-type="blockOuter" data-id="6"><div class="bn-block" data-node-type="blockContainer" data-id="6">Column Paragraph 2</div></div><div class="bn-block-outer" data-node-type="blockOuter" data-id="7"><div class="bn-block" data-node-type="blockContainer" data-id="7">Column Paragraph 3</div></div>
```

Comment:
I see that even though it's external HTML, the block structure (`blockOuter`, `blockContainer`, etc) divs are still preserved. I guess this is ok for now?